datanode/networkhistory/ipfs/migration.go

```diff
@@ -77,11 +56,76 @@ func MigrateIpfsStorageVersion(log *logging.Logger, ipfsDir string) error {
 		return fmt.Errorf("failed to find local ipfs directory: %w", err)
 	}
 
-	fetcher := createFetcher("", localIpfsDir)
-	err = migrations.RunMigration(context.Background(), fetcher, latestSupportedVersion(), localIpfsDir, false)
-	if err != nil {
+	// fetcher := createFetcher("", localIpfsDir)
+	// err = migrations.RunMigration(context.Background(), fetcher, latestSupportedVersion(), localIpfsDir, false)
```

Comment:
Can we just delete these if we don't need them rather than comment them out? If we need to revert they will be in the git history anyway

Reply:
Ohh yess, ofcourse I left it but removed all the functions related to them.